docs/other/official-links.md

```diff
@@ -16,7 +16,8 @@ This page contains all the official links for the Status Network ecosystem.
 ## Network Tools
 - **Bridge**: [bridge.status.network](https://bridge.status.network)
 - **Block Explorer**: [sepoliascan.status.network](https://sepoliascan.status.network)
-- **RPC Endpoint**: `https://public.sepolia.rpc.status.network`
+- **RPC Endpoint**: [public.sepolia.rpc.status.network](https://public.sepolia.rpc.status.network)
```

Comment:
I don't think hyperlinking is helpful for rpc. Something like one-click copy is more useful.